docs/guide/env-and-mode.md

```diff
@@ -24,7 +24,7 @@ It will also replace these strings appearing in JavaScript strings and Vue templ
 
 ## `.env` Files
 
-Vite uses [dotenv](https://github.com/motdotla/dotenv) to load additional environment variables from the following files in your [environment directory](/config/#envdir):
+Vite uses [dotenv](https://github.com/motdotla/dotenv) to load additional environment variables from the following files in your [environment directory](/config/#envdir) (if define the same variables will override variables in the following reverse order):
```

Comment:
IMO it reads hard :thinking: 
Maybe better something like this? Could even be written better.

```suggestion
Vite uses [dotenv](https://github.com/motdotla/dotenv) to load additional environment variables from the following files in your [environment directory](/config/#envdir) (if they define the same variable, the variable defined in the most specific file will win):
```

Reply:
I also think `.env.${mode}.local` is in the most specific, but it will be overwrite 😅

Reply:
in not the most specific will win?

Reply:
If the most specific currently don't win, then IMO this is a bug and should be changed
And I assume this is what #6306 is fixing :thinking:

Reply:
check https://cli.vuejs.org/guide/mode-and-env.html#environment-variables, see green alert with title `Env Loading Priorities`